core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -133,7 +133,8 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
   this.serverConfig.setProperty(KafkaConfig.AuthorizerClassNameProp, classOf[SimpleAclAuthorizer].getName)
   // Some needed configuration for brokers, producers, and consumers
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicPartitionsProp, "1")
-  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "1")
+  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
+  this.serverConfig.setProperty(KafkaConfig.OffsetCommitRequiredAcksProp, "1")
```

Comment:
@ijuma I found why - at test startup, 
before the admin user is granted read access on all topics,
ReplicaFetcherThread encounters an auth error.

``` 
Adding ACLs for resource `Cluster:kafka-cluster`: 
 	User:admin has Allow permission for operations: ClusterAction from hosts: * 

Current ACLs for resource `Cluster:kafka-cluster`: 
 	User:admin has Allow permission for operations: ClusterAction from hosts: * 

[2016-05-24 17:22:51,939] ERROR [ReplicaFetcherThread-0-0], Error for partition [__consumer_offsets,0] to broker 0:org.apache.kafka.common.errors.TopicAuthorizationException: Not authorized to access topics: [Topic authorization failed.] (kafka.server.ReplicaFetcherThread:97)
[2016-05-24 17:22:51,939] ERROR [ReplicaFetcherThread-0-0], Error for partition [__consumer_offsets,0] to broker 0:org.apache.kafka.common.errors.TopicAuthorizationException: Not authorized to access topics: [Topic authorization failed.] (kafka.server.ReplicaFetcherThread:97)
Adding ACLs for resource `Topic:*`: 
 	User:admin has Allow permission for operations: Read from hosts: * 
```